src/pages/PreviousRuns.js

```diff
@@ -1,8 +1,244 @@
+      submissionTimestamp: '2022-01-27T22:27:15.591Z'
+    },
+    {
+      workflowName: '11-mock-abc-1',
+      workflowInputs: '',
+      workflowStatus: 'Failed',
+      workflowUrl: 'https://raw.githubusercontent.com/abc/xyz',
+      submissionTimestamp: '2022-07-14T22:22:15.591Z'
+    },
+    {
+      workflowName: '12-dockstore-abc-hello-1',
+      workflowStatus: 'Done',
+      workflowInputs: '{ "hello.inputs_array": [12-1, 2, 3, 4, 5] }',
+      workflowUrl: 'https://dockstore/abc/hello.wdl',
+      submissionTimestamp: '2022-07-12T22:57:15.591Z'
+    },
+    {
+      workflowName: '13-simple-hello-world-2',
+      workflowStatus: 'Done',
+      workflowInputs: '{ "wf_hello.hello.addressee": "13-World" }',
+      workflowUrl: 'https://raw.githubusercontent.com/broadinstitute/cromwell/develop/centaur/src/main/resources/standardTestCases/hello/hello.wdl',
+      submissionTimestamp: '2021-01-27T22:28:15.591Z'
+    },
+    {
+      workflowName: '14-mock-abc-2',
+      workflowStatus: 'Submitted',
+      workflowInputs: '',
+      workflowUrl: 'https://raw.githubusercontent.com/abc/xyz',
+      submissionTimestamp: '2022-07-14T22:28:15.591Z'
+    },
+    {
+      workflowName: '15-dockstore-abc-hello-1',
+      workflowStatus: 'Done',
+      workflowInputs: '{ "hello.inputs_array": [15-1, 2, 3, 4, 5] }',
+      workflowUrl: 'https://dockstore/abc/hello.wdl',
+      submissionTimestamp: '2021-05-12T22:26:15.591Z'
+    },
+    {
+      workflowName: '16-simple-hello-world-3',
+      workflowStatus: 'Aborted',
+      workflowInputs: '{ "wf_hello.hello.addressee": "16-World" }',
+      workflowUrl: 'https://raw.githubusercontent.com/broadinstitute/cromwell/develop/centaur/src/main/resources/standardTestCases/hello/hello.wdl',
+      submissionTimestamp: '2022-01-27T22:29:15.591Z'
+    },
+    {
+      workflowName: '17-mock-abc-3',
+      workflowStatus: 'Failed',
+      workflowInputs: '',
+      workflowUrl: 'https://raw.githubusercontent.com/abc/xyz',
+      submissionTimestamp: '2019-10-20T22:22:15.591Z'
+    },
+    {
+      workflowName: '18-dockstore-abc-hello-1',
+      workflowStatus: 'Done',
+      workflowInputs: '{ "hello.inputs_array": [18-1, 2, 3, 4, 5] }',
+      workflowUrl: 'https://dockstore/abc/hello.wdl',
+      submissionTimestamp: '2019-07-12T22:24:15.591Z'
+    }
+  ]
+
+  const sortedPreviousRuns = _.orderBy(sort.field, sort.direction, previousRuns)
+
+  const firstPageIndex = (pageNumber - 1) * itemsPerPage
+  const lastPageIndex = firstPageIndex + itemsPerPage
+  const paginatedPreviousRuns = sortedPreviousRuns.slice(firstPageIndex, lastPageIndex)
+
+
+  return h(Fragment, [
+    headerBar(),
+    div({ style: { margin: '4em' } }, [
+      h2(['Previous Runs']),
+      div(['Remember to turn off your Cromwell App in Terra once you are done to prevent incurring costs.']),
```

Comment:
Can this be standardized or extracted? It feels like it could become quickly tedious to copy/paste it, and even more tedious to keep it all updated if it ever changes?

Reply:
Actually, I am thinking that does it make sense to have it here? Maybe it should be always at the footer of the page as remainder or it could be added to the Create Cloud Environments component in Terra UI? What do you think?

Reply:
Checked it with Tiffany. There is a plan to add this warning in the Create Cloud Environments component in Terra UI. She was also going to look into adding this warning as a footer in the app. If that is the case, we might have a ticket later on to add this message to the footer and it can be removed from here. But I will leave this here for now.